src/v2/stack_aggregator.py

```diff
@@ -310,7 +323,14 @@ def get_result(self) -> Union[StackAggregatorResultForFreeTier,
         # denormalize package details according to request.dependencies relations
         package_details = self._get_denormalized_package_details()
         unknown_dependencies = self._get_direct_unknown_packages()
+        started_at = time.time()
+
         license_analysis = get_license_analysis_for_stack(package_details)
+
+        eri = self._request.external_request_id if self._request is not None else 'UNKNOWN'
```

Comment:
Same here.

Reply:
Same as above, added for unit test cases.